### PR TITLE
Add testrunner for full gardener lifecycle tests and remove deprecated kibana url

### PR DESCRIPTION
**Pull request by schrodit**
**What this PR does / why we need it**:
Added a testrunner for gardener lifecycle tests which means that we do not need dynamic parameters like cloudprovider, k8s versions etcd in the first step.
Maybe we will need this for later easier testing but for now the lifecycle is hardcoded.

Therefore the only thing we need for this gardener tempalting is 2 component descriptors: 1 fomr the latest stable release and one from the current release.

- removed deprecated kibana field

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator
Added a testrunner for gardener lifecycle tests
```

### CODE REVIEW

File: pkg/testrunner/template/gardener.go [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+	"fmt"
+	"github.com/gardener/gardener/pkg/chartrenderer"
+	"github.com/gardener/gardener/pkg/client/kubernetes"
+	"github.com/gardener/test-infra/pkg/testrunner"
+	"github.com/gardener/test-infra/pkg/testrunner/componentdescriptor"
+	"github.com/gardener/test-infra/pkg/util"
+	log "github.com/sirupsen/logrus"
+)
+
+// RenderShootTestrun renders a helm chart with containing testruns, adds the provided parameters and values, and returns the parsed and modified testruns.
```
Comment (OlegLoewen) [inline pkg/testrunner/template/gardener.go]: Update comment/docu

File: pkg/util/util.go [hunk 2]
```diff
@@ -192,3 +193,12 @@ func StringArrayContains(ar []string, elem string) bool {
 	}
 	return false
 }
+
+// String default checks of a string is defined.
```
Comment (dguendisch) [inline pkg/util/util.go]:
```suggestion
// StringDefault checks if a string is defined.
```

File: pkg/util/util.go [hunk 3]
```diff
@@ -184,6 +184,7 @@ func MarshalNoHTMLEscape(v interface{}) ([]byte, error) {
 	return buffer.Bytes(), nil
 }
 
+// StringArrayContains checks if a string array contains s string element
```
Comment (dguendisch) [inline pkg/util/util.go]:
```suggestion
// StringArrayContains checks if a string array contains the string elem
```

File: pkg/testrunner/componentdescriptor/componentdescriptor.go [hunk 4]
```diff
@@ -37,6 +40,18 @@ func GetComponents(content []byte) (ComponentList, error) {
...
 }
 
+// GetComponentsFromFile read a component descriptor and returns a ComponentList
+func GetComponentsFromFile(file string) (ComponentList, error) {
+	if file == "" {
+		return make(ComponentList, 0), nil
+	}
+	data, err := ioutil.ReadFile(file)
+	if err != nil {
+		return nil, fmt.Errorf("Cannot read component descriptor file %s: %s", file, err.Error())
```
Comment (dguendisch) [inline pkg/testrunner/componentdescriptor/componentdescriptor.go]:
```suggestion
		return nil, fmt.Errorf("cannot read component descriptor file %s: %s", file, err.Error())
```

**Pull request merged**